command/connect/envoy/bootstrap_tpl.go

```diff
@@ -27,6 +27,11 @@ type BootstrapTplArgs struct {
 	// TLS is enabled.
 	AgentCAFile string
 
+	// AdminAccessLogPath The path to write the access log for the
+	// administration server. If no access log is desired specify
+	// "/dev/null".
```

Comment:
Should we just make /dev/null be the default so specifying empty is also valid? This is kinda internal only but we do a bunch of default setting before the template already so this seems like a reasonable approach here too.

Reply:
I made it so that it can also be left empty and then the default would be picked up.